two_factor/models.py

```diff
@@ -53,26 +63,16 @@ def key_validator(*args, **kwargs):
     return hex_validator()(*args, **kwargs)
 
 
-class PhoneDevice(ThrottlingMixin, Device):
-    """
-    Model with phone number and token seed linked to a user.
-    """
+class TwoFactorMixin(models.Model):
     class Meta:
         app_label = 'two_factor'
+        abstract = True
 
-    number = PhoneNumberField()
+    drift_range = ()
```

Comment:
`drift_range` should be set to `None` here. That way if someone forgets to set it, they'll get an error.